### PR TITLE
Global steam client link

### PR DESCRIPTION
**Pull request by Revadike**
Resolves https://github.com/tfedor/AugmentedSteam/issues/399

### CODE REVIEW

**Comment by tfedor** [comment]
When would this be useful?

**Comment by Revadike** [comment]
> When would this be useful?

For example:
If you have a new chat message notification, I'd prefer to open it with steam client, rather than the web chat. This is also consistent with the friend profile chat dropdown options.
There are other scenarios when steam client would be preferred.
People without the steam client installed, can simply turn this feature off.

**Comment by MxtOUT** [comment]
> If you have a new chat message notification, I'd prefer to open it with steam client
Why?
Especially if we resolve #391, there's at least one reason more to stay in the browser

> There are other scenarios when steam client would be preferred.
And they would be?

**Comment by Revadike** [comment]
I just prefer the client chat over web chat. It's personal preference. I'm sure I won't be the only one. Also, it's a quick way to switched to "logged in" page (if you were logged out), or a AS-less page. I dunno, I'm sure there are many other reasons I can't think of right now.

**Comment by MxtOUT** [comment]
I actually also only chat via the client but that's because I'm directly clicking the message popups and I never use the web page for this, so I can't say anything about this.

I'm not convinced though

**Comment by tfedor** [comment]
If the main (and seems like only) purpose of this feature is to open chat in client, it seems like shooting out of cannon into sparrows

**Comment by Revadike** [comment]
> If the main (and seems like only) purpose of this feature is to open chat in client, it seems like shooting out of cannon into sparrows

Wrong. There are plenty of use cases:
 * Easily switch sessions, if you are logged into steam client with a different account than browser
 * Easily disable/remove augmented steam and any other addons
 * Easily login to steam, provided you're logged into steam client and not in browser
 * An easy way to open/launch the steam client
 * An easy way to navigate to the same page as your browser in the steam client
 * Avoid using web chat in any way
And there are probably many other cases I simply can't think of right now.
Giving people at least the option will be positive.

**Comment by tfedor** [comment]
Ok, I have a better solution that I would be fine with. We have an option to remove "Install Steam" button. We could add another option, that would instead of hiding it, replaced it with "Open in client" button. I would be ok with that implementation, instead of having link in menu.

File: js/content/common.js [hunk 1]
```diff
@@ -951,10 +951,23 @@ let EnhancedSteam = (function() {
         Promise.reject(err);
     };
 
+    self.viewInSteamButton = function() {
+        if (!SyncedStorage.get("showclient")) { return; }
+
+        let btn = document.querySelector("div.header_installsteam_btn > a");
+        btn.textContent = Localization.str.viewinclient;
+        btn.href =  `steam://openurl/${window.location.href}`;
+        btn.style.backgroundImage = "url(https://steamstore-a.akamaihd.net/public/images/v6/icon_platform_linux.png)";
```
[the remark this answers is not in the document]
Comment (Revadike) [reply js/content/common.js]: Resolved in https://github.com/tfedor/AugmentedSteam/pull/489/commits/2f565ca7a2e9167fda67d1ef4ec612d53751546d

**Comment by MxtOUT** [comment]
This was properly merged, but due to a GitHub bug it doesn't show as such.
See https://github.com/tfedor/AugmentedSteam/commit/f5bba2ee6a0e68b070ba414f9ba5d71c65f6c35c for proof

**Pull request closed without merging**